src/api/models/Vote.ts

```diff
@@ -47,6 +47,26 @@ export class Vote extends Bookshelf.Model<Vote> {
         }
     }
 
+    public static async fetchAllFromMeByProposalId(proposalId: number, withRelated: boolean = true): Promise<Collection<Vote>> {
```

Comment:
what does this mean? "FromMe"? the functionality seems to implement fetchAllByProposal.